tour_model_eval/trips_in_bins_and_clusters.ipynb

```diff
@@ -130,138 +140,184 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "latest-cooling",
+   "id": "systematic-huntington",
    "metadata": {
     "scrolled": false
    },
    "outputs": [],
    "source": [
-    "bin_start_lat = []\n",
-    "bin_start_log = []\n",
-    "\n",
-    "# Flip indices because points are in geojson (i.e. lon, lat)\n",
-    "for trip in bin_trips:\n",
-    "    trip_start_lat = trip.data.start_loc[\"coordinates\"][1]\n",
-    "    trip_start_log = trip.data.start_loc[\"coordinates\"][0]\n",
-    "    bin_start_lat.append(trip_start_lat)\n",
-    "    bin_start_log.append(trip_start_log)"
+    "# bin_trips"
```

Comment:
why do you have a cell whose entire code is just a commented out line for `bin_trips`?

Reply:
Oh, codes that are commented out are the way I try to see the output of the result like`bin_trips`,`bins` etc. They don't really necessary in the code.

Reply:
In that case, you should remove them from the notebook before submitting for review.